robotidy/transformers/RenameVariables.py

```diff
@@ -0,0 +1,420 @@
+import re
+from typing import List, Pattern, Tuple
+
+from robot.api.parsing import Arguments, Token
+from robot.errors import VariableError
+from robot.variables import VariableIterator
+
+try:  # only available starting RF 6.0
+    from robot.variables.search import _search_variable
```

Comment:
Now I can see I could just use search_variables and it will work for all versions without need for extra tricks. I will update it tomorrow